src/app/pages/select-network/components/network-list-item.layout.tsx

```diff
@@ -37,7 +37,12 @@ export function NetworkListItemLayout(props: NetworkListItemLayoutProps) {
       data-testid={SettingsSelectors.NetworkListItem}
       {...rest}
     >
-      <Flex width="100%" justifyContent="space-between" alignItems="center">
+      <Flex
+        width="100%"
+        justifyContent="space-between"
+        alignItems="center"
+        data-testid={network.testId || ''}
```

Comment:
Seems `testId` is type `?string`, is the `|| ''` part necessary?


```suggestion
        data-testid={network.testId}
```

Reply:
I just thought `data-testid=""` is prob better than `data-testid="undefined"`, but don't have strong opinion here

Reply:
If it's undefined, won't it be `data-testid={undefined}`, meaning the attribute is completely omitted? We probably want to hide the attribute if it's not being used rather than showing it with an empty string no?

Reply:
not sure here, I think its probably more consistent to show them on all list items, even if they are empty string

Reply:
Alright, your call, I just thought it looked weird, but don't have a rule or a convincing argument to back me up 😂